test/test-functions

```diff
@@ -894,6 +895,15 @@ printf "[Unit]\nConditionVirtualization=container\n\n[Service]\nTimeoutSec=240s\
 mkdir -p /etc/systemd/system/systemd-journal-flush.service.d
 printf "[Service]\nTimeoutSec=180s\n" >/etc/systemd/system/systemd-journal-flush.service.d/timeout.conf
 
+# With the initrd -> asan_wrapper -> systemd transition we might end up with
+# plymouthd still running, which will eat our inputs and make debugging via tty
+# impossible, so let's kill it before exec()'ing systemd.
+# Note: we can't use pkill/pidof/etc. here due to a bug in libasan, see:
+#   - https://github.com/llvm/llvm-project/issues/49223
+#   - https://bugzilla.redhat.com/show_bug.cgi?id=2098125
+# Thankfully, the standalone kill binary (not the bash's built-in) works too.
+/bin/kill --all --verbose plymouthd || :
```

Comment:
on debian:

```
-bash-5.1# /bin/kill --all --verbose plymouthd || :
kill: invalid argument 

Usage:
 kill [options] <pid> [...]

Options:
 <pid> [...]            send signal to every <pid> listed
 -<signal>, -s, --signal <signal>
                        specify the <signal> to be sent
 -q, --queue <value>    integer value to be sent with the signal
 -l, --list=[<signal>]  list all signal names, or convert one to a name
 -L, --table            list all signal names in a nice table

 -h, --help     display this help and exit
 -V, --version  output version information and exit

For more details see kill(1).
```

Reply:
good news is, killing plymouthd indeed fixes the problem I have, even without sanitizers! halleluja!

Reply:
> on debian:
> 
> ```
> -bash-5.1# /bin/kill --all --verbose plymouthd || :
> kill: invalid argument 
> 
> Usage:
>  kill [options] <pid> [...]
> 
> Options:
>  <pid> [...]            send signal to every <pid> listed
>  -<signal>, -s, --signal <signal>
>                         specify the <signal> to be sent
>  -q, --queue <value>    integer value to be sent with the signal
>  -l, --list=[<signal>]  list all signal names, or convert one to a name
>  -L, --table            list all signal names in a nice table
> 
>  -h, --help     display this help and exit
>  -V, --version  output version information and exit
> 
> For more details see kill(1).
> ```

*[screams internally]*

Looks like `killall` works too, at least on Fedora.